config/jobs/kubernetes-sigs/reference-docs/referencedocs-presubmit-master.yaml

```diff
@@ -0,0 +1,31 @@
+presubmits:
+  kubernetes-sigs/reference-docs:
+  - name: pull-referencedocs-build
+    decorate: true
+    always_run: true
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: golang:1.15
+        command:
+        - mkdir -p ${BUILD_PATH}/src/k8s.io
+        - wget https://github.com/kubernetes/kubernetes/archive/v${KUBE_VERSION}.0.tar.gz -O ${BUILD_PATH}/src/k8s.io/kubernetes-src.tar.gz
```

Comment:
This is my first attempt to write a job.
Is it possible to run a series of commands or should I write a script that creates the installation and calls the
make targets?